src/data/pools/poolData.ts

```diff
@@ -1,12 +1,14 @@
 import { useQuery } from '@apollo/client'
 import gql from 'graphql-tag'
-import { useDeltaTimestamps } from 'utils/queries'
-import { useBlocksFromTimestamps } from 'hooks/useBlocksFromTimestamps'
+import { getDeltaTimestamps, useDeltaTimestamps } from 'utils/queries'
+import { getBlocksFromTimestamps, useBlocksFromTimestamps } from 'hooks/useBlocksFromTimestamps'
 import { PoolData } from 'state/pools/reducer'
 import { get2DayChange } from 'utils/data'
 import { formatTokenName, formatTokenSymbol } from 'utils/tokens'
 import { useActiveNetworks, useClients } from 'state/application/hooks'
 import { FEE_BASE_UNITS } from 'utils'
```

Comment:
remove these line b/c of unused code
![image](https://user-images.githubusercontent.com/23650671/181256227-96551f32-89d2-4cb4-ba1a-82a121410912.png)